src/app/Rules/CreateRule.tsx

```diff
@@ -101,6 +101,15 @@ const Comp: React.FC = () => {
   const [errorMessage, setErrorMessage] = React.useState('');
   const [loading, setLoading] = React.useState(false);
 
+  const targetSubject = React.useRef(new BehaviorSubject(NO_TARGET)).current;
```

Comment:
I'm wondering what was the reason for using this vs the old `context.target.target()`?

Reply:
The `context.target.target` is for global target selection. Here, we just want to see detail without changing that. 

Also, the reason behind using a subject is to ensure there is a single observable to subscribe to to its entire lifecycle. And we can control to ignore previous inner observable when the target node change.

Reply:
Ah, I see thanks for that.